src/reflex.js

```diff
@@ -1,30 +1,38 @@
 // @flow
-import { type Props } from './withBase'
-import * as srcUtils from './utils'
+import * as _utils from './utils'
 
-export type GridProps = Props
-
-export { default as withGrid } from './withGrid'
-export { default as withBase } from './withBase'
-export { default as ConfigProvider } from './configProvider'
+// Core
 export { default as Base } from './base'
 export { default as Grid } from './grid'
 export { default as Layout } from './layout'
-export { version } from '../version'
-export const utils = srcUtils
+export { default as ConfigProvider } from './configProvider'
 
+// Derived
 export { default as Col } from './derived/col'
 export { default as Root } from './derived/root'
 export { default as Offset } from './derived/offset'
+
+// HOC
+export { default as withGrid } from './withGrid'
+export { default as withBase } from './withBase'
+export { default as withResolution } from './withResolution'
+
+// Utils / Dev
+export const utils = _utils
+export { version } from '../version'
 export { default as initDevMode } from './dev'
 
+// Types
 export type {
-  Size,
   AlignGrid,
-  Justify,
+  Dev,
+  DisplayValues,
   Fixed,
-  Overflow,
+  GridProps,
   Height,
-  Dev,
+  Justify,
+  Overflow,
+  Size,
   SpacingProps,
+  SpacingValues,
 } from './types'
```

Comment:
I grouped the exports and moved `GridProps` to types now that it's gotten a bit more complex